sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/StatusData.java

```diff
@@ -38,8 +39,8 @@ static StatusData error() {
    * Returns a {@link StatusData} with the given {@code code} and {@code description}. If {@code
    * description} is {@code null}, the returned {@link StatusData} does not have a description.
    */
-  static StatusData create(StatusCode code, String description) {
-    return ImmutableStatusData.create(code, description);
+  static StatusData create(StatusCode code, @Nullable String description) {
```

Comment:
I just added Nullable to the signature since we document it in the javadoc. I'm not sure why jamp isn't showing a diff for this

Reply:
Perhaps jcmp is smarter than expected? Probably not, but anyways just one thing to keep in mind is that adding `@Nullable` to a return type can cause kotlin code to stop compiling, while adding to a parameter cannot, so former is technically not semver compatible and later is (if we actually considered kotlin compatibility when defining our semver :P)

Reply:
I bet jamicmp just isn't smart enough to look at parameter annotations.